packages/forms-web-app/src/middleware/check-path-allowed.js

```diff
@@ -9,6 +9,10 @@ const checkPathAllowed = (req, res, next) => {
 		'/before-you-start/use-existing-service-local-planning-department',
 		'/appellant-submission/submission-information',
 		'/full-appeal/submit-appeal/declaration-information',
+		'/full-appeal/submit-appeal/code-expired',
+		'/full-appeal/submit-appeal/request-new-code',
+		'/appeal-householder-decision/code-expired',
+		'/appeal-householder-decision/request-new-code',
```

Comment:
For consistency, should these URL exemptions not be handled using the `skipMiddlewareForPaths` middleware in the same way as the enter code pages?

Reply:
possibly - that middleware is used for `enter-code` because the url is different depending on id, whereas these urls - like the others in the list - are fixed.